OpenEXR/IlmImf/ImfMultiPartOutputFile.cpp

```diff
@@ -498,7 +498,7 @@ MultiPartOutputFile::Data::writeChunkTableOffsets (vector<OutputPartData*> &part
 
         Int64 pos = os->tellp();
 
-        if (pos == -1)
+        if (pos == Int64(-1))
```

Comment:
shouldn't these all be pos == -1ll

Reply:
I think the checks are correct, if the stream is truly a stream, you can't do what this code wants, and throws, right?